templates_jinja2/team_details.html

```diff
@@ -142,6 +142,8 @@ <h3><a href="/event/{{ comp.event.key_name }}">{{ comp.event.name }}</a></h3>
                   </ul>
                   {% endif %}
                 </p>
+                <a class="btn btn-default" href="https://www.youtube.com/watch_videos?video_ids={{comp.video_ids}}" target="_blank">
+                  <span class="glyphicon glyphicon-play-circle"></span> Watch Me at This Event</a>
```

Comment:
Copy nit: "Watch All Matches" or something, we don't really use "me" anywhere else on the site like this

Reply:
Do you think the button location is sufficient to scope "All" to "This this team at this event"? Or should we be more specific with something like "Watch all of ~team number~'s matches"?

Reply:
"Watch All" should be sufficient, I think

Reply:
Sounds fine to me